### DISCUSSION TITLE
Review of Thesaurus

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#239] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/thesaurus.svg">

**Comment by michael2063** [comment]
Unfortunately, it cannot be used in languages other than English, contrary to what is stated in the plugin management section. As a result, it cannot be used for any other languages.

**Comment by mpazbedoya-create** [comment]
Sólo funciona para el idioma Ingles.
